ts/src/management/tunnelManagementHttpClient.ts

```diff
@@ -38,6 +38,9 @@ const clustersApiPath = '/clusters';
 const tunnelAuthentication = 'Authorization';
 const checkAvailablePath = ':checkNameAvailability';
 const createNameRetries = 3;
+enum ManagementApiVersions {
```

Comment:
This should probably be exported so consumers can use the enum value instead of having to use the string

Reply:
Updated to export it